src/dune_engine/action_exec.mli

```diff
@@ -1,29 +1,12 @@
 open Import
 
-(** Type for dependency requested by the dynamic action.
-
-    Must be different from [Dep.t] type because we require it to be
-    marshallable. *)
```

Comment:
Was this comment stale? I'm confused because it claimed we need something special compared to `Dep.t` but I can't spot where this special ability was used.

Reply:
Yes everything is marshallable ever since we got rid of the damned predicates 🙂

Reply:
Oh, right, cool :)